common/tools/eslint-plugin-azure-sdk/package.json

```diff
@@ -44,6 +44,9 @@
     "lint": "eslint src tests --ext .ts",
     "pack": "npm pack 2>&1",
     "prebuild": "npm run clean",
+    "integration-test:browser": "echo skipped",
+    "integration-test:node": "echo skipped",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
```

Comment:
Why are we adding this now? Both are skipped r8?

Reply:
Without this line rush complains that the task doesn't exist (because for some reason our live test tasks run live tests of all dependencies too)